lib/fizzy/execute.cpp

```diff
@@ -1521,12 +1521,26 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, span<const Value>
             break;
         }
 
+        case Instr::f32_abs:
+        {
+            // TODO: This can be optimized https://godbolt.org/z/aPqvfo
+            unary_op(stack, static_cast<float (*)(float)>(std::fabs));
```

Comment:
`std::fabs` is unfortunately not a template, but several overloaded variants.

Reply:
Also would work without casts: `fabsf` and `fabs` C functions.

Reply:
> (Also would work without casts: `fabsf` and `fabs` C functions.)

That would only work for `fabsf`...